### PR TITLE
Migrate to json for sig-transaction membership

### PR DESCRIPTION
**Pull request by andylokandy**
Signed-off-by: andylokandy <andylokandy@hotmail.com>

### CODE REVIEW

**Comment by Rustin-Liu** [comment]
@andylokandy Could you please add these emails and companies? Thanks for your support.
```json
[
  {
    githubId: 'AndreMouche',
    level: 'committer',
    email: 'wuxuelian@pingcap.com',
    company: 'pingcap_Leader'
  },
  {
    githubId: 'bobotu',
    level: 'committer',
    email: 'lizejun@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'cfzjywxk',
    level: 'committer',
    email: 'xurui@pingcap.com',
    company: 'pingcap'
  },
  {
    githubId: 'disksing',
    level: 'committer',
    email: 'menglong@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'jackysp',
    level: 'committer',
    email: 'yusp@pingcap.com',
    company: 'pingcap_unknow'
  },
  {
    githubId: 'lysu',
    level: 'committer',
    email: 'suli@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'sticnarf',
    level: 'committer',
    email: 'chenyilin@pingcap.com',
    company: 'pingcap_R&D_intern'
  },
  {
    githubId: 'tiancaiamao',
    level: 'committer',
    email: 'mkl@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'youjiali1995',
    level: 'committer',
    email: 'zhaolei@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'coocood',
    level: 'committer',
    email: 'z@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'imtbkcat',
    level: 'committer',
    email: 'songlingyu@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'MyonKeminta',
    level: 'committer',
    email: 'zhouzhenjing@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'nrc',
    level: 'leader',
    email: 'nick@pingcap.com',
    company: 'pingcap_R&D'
  },
  {
    githubId: 'andylokandy',
    level: 'leader',
    email: 'luodian@pingcap.com',
    company: 'pingcap_R&D_intern'
  },
  {
    githubId: 'zhangjinpeng1987',
    level: 'committer',
    email: 'zhangjinpeng@pingcap.com',
    company: 'pingcap_unknow'
  }
]
```

File: sig/transaction/membership.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "transaction",
+  "techLeaders": [
+    {
+      "githubId": "nrc"
```
Comment (Rustin-Liu) [inline sig/transaction/membership.json:5]: This should be `githubName`: https://github.com/tidb-community-bots/ti-community-bot/pull/75

**Comment by Rustin-Liu** [comment]
[Doc for lgtm and megre plugins](http://prow.tidb.io/plugins?repo=tikv%2Fcommunity).

Hope it helps you.

**Review by Rustin-Liu — APPROVED** [review]

**Review by AndreMouche — APPROVED** [review]

**Comment by sticnarf** [comment]
@andylokandy Now the link here https://github.com/tikv/community/tree/master/sig/transaction#members is broken.

**Comment by sticnarf** [comment]
Will there be a static page that will dynamically pull the json and render a human-friendly list? Maybe it could be put somewhere on tikv.github.io

**Comment by Rustin-Liu** [comment]
> Will there be a static page that will dynamically pull the json and render a human-friendly list? Maybe it could be put somewhere on tikv.github.io

@andylokandy working on this.

**Comment by andylokandy** [comment]
> Will there be a static page that will dynamically pull the json and render a human-friendly list? Maybe it could be put somewhere on tikv.github.io

It's now here:
https://tikv.github.io/members/build/index.html?name=transaction

**Pull request merged**